### DISCUSSION TITLE
Where to Report Issues

### DISCUSSION THREAD

**Discussion by luziusmeisser** [Frankencoin-ZCHF/Frankencoin#22] [Ideas]
Technical issues and ideas can be directly reported to the issue tracking system of the git repository.

For frontend ideas, this [can be found here](https://gitlab.com/frankencoin-zchf/frankencoin-dapp/-/issues).

For ideas on how to improve the smart contracts, [go here](https://github.com/Frankencoin-ZCHF/FrankenCoin/issues).

**Comment by lclc** [comment]
Frontend seems to have been moved from GitLab (currently linked) to GitHub (here https://github.com/Frankencoin-ZCHF/frankencoin-dapp/issues )
